From fe09305eb6f907e4eb637b8edd0c8a986187d1dd Mon Sep 17 00:00:00 2001
From: Rosen Penev <rosenp@gmail.com>
Date: Sat, 8 Jun 2024 15:23:51 -0700
Subject: [PATCH] fix crash under big endian musl

iperf_printf is using an int format here but an int64_t variable. The format only needs the first 3 digits. Cast to int to fix it.
---
 src/iperf_api.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/iperf_api.c
+++ b/src/iperf_api.c
@@ -4056,7 +4056,7 @@ iperf_print_results(struct iperf_test *t
                                 iperf_printf(test, report_sender_not_available_summary_format, "SUM");
                         }
                         else {
-                          iperf_printf(test, report_sum_bw_retrans_format, mbuf, start_time, sender_time, ubuf, nbuf, total_retransmits, report_sender);
+                          iperf_printf(test, report_sum_bw_retrans_format, mbuf, start_time, sender_time, ubuf, nbuf, (int)total_retransmits, report_sender);
                         }
                 } else {
                     /* Summary sum, TCP without retransmits. */
